appveyor.yml

```diff
@@ -1,20 +1,38 @@
+# Notes:
+#   - Minimal appveyor.yml file is an empty file. All sections are optional.
+#   - Indent each level of configuration with 2 spaces. Do not use tabs!
+#   - All section names are case-sensitive.
+#   - Section names should be unique on each level.
+
 version: "{build}"
+
 os: Windows Server 2012 R2
+
+clone_folder: c:\go\src\github.com\ipfs\go-ipfs
+
 environment:
   GOPATH: c:\gopath
   TEST_NO_FUSE: 1
   TEST_VERBOSE: 1
   TEST_SUITE: test_sharness_expensive
+  matrix:
+  - GOARCH: amd64
+    GOVERSION: 1.5.1
+    GOROOT: c:\go
+    DOWNLOADPLATFORM: "x64"
+  
 install:
-  - set PATH=%GOPATH%\bin;c:\go\bin;%PATH%
-  - rmdir c:\go /s /q
-  - appveyor DownloadFile https://storage.googleapis.com/golang/go1.5.1.windows-amd64.msi
-  - msiexec /i go1.5.1.windows-amd64.msi /q
+  # Enable make
+  - choco install mingw
+  - SET PATH=c:\tools\mingw64\bin;%PATH%
   - go version
   - go env
+  
 build_script:
-  - make install
+  - mingw32-make install
```

Comment:
From the [doc](https://github.com/ipfs/go-ipfs/blob/master/docs/windows.md#L8), it appears this should have been `mingw32-make nofuse`.
An os detect could have been incorporated into the makefile.